gemsupervisor/src/common/GEMSupervisor.cc

```diff
@@ -1176,7 +1292,7 @@ class InitCompare {
     else if (classname == "tcds::ici::ICIController")               priority = 101; // must be second of TCDS!
     else if (classname == "tcds::pi::PIController")                 priority = 100; // must be first!?
     else if (classname == "gem::hw::amc13::AMC13Manager")           priority =  95; // after AMCManagers but not last
-    else if (classname == "gem::hw::glib::GLIBManager")             priority =  90; // before (or simultaneous with) OH manager and before AMC13
+    else if (classname == "gem::hw::amc::AMCManager")               priority =  90; // before (or simultaneous with) OH manager and before AMC13
```

Comment:
Could have been removed, the `gem::hw::amc::AMCManager` class is present 2 lines below.